dags/test_dag.py

```diff
@@ -1,43 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
@rhwang10 isn't this file part of the official Apache project?

Reply:
Oops, you're right